src/os/windows/win32.cpp

```diff
@@ -713,17 +690,18 @@ int OTTDStringCompare(const char *s1, const char *s2)
  */
 bool IsWindowsVistaOrGreater()
 {
+	static DllLoader _kernel32(L"kernel32.dll");
 	typedef BOOL (WINAPI * LPVERIFYVERSIONINFO)(LPOSVERSIONINFOEX, DWORD, DWORDLONG);
 	typedef ULONGLONG (NTAPI * LPVERSETCONDITIONMASK)(ULONGLONG, DWORD, BYTE);
 #ifdef UNICODE
-	static LPVERIFYVERSIONINFO _VerifyVersionInfo = (LPVERIFYVERSIONINFO)GetProcAddress(GetModuleHandle(_T("Kernel32")), "VerifyVersionInfoW");
+	static LPVERIFYVERSIONINFO _VerifyVersionInfo = _kernel32.GetProcAddress("VerifyVersionInfoW");
```

Comment:
Given we have already removed Win9x-specific code, and VerifyVersionInfoW and VerSetConditionMask were introduced in Windows 2000, we should be safe to simply refer to these directly instead of dynamically loading them. I think we also assume UNICODE now too, so can get rid of the ANSI version.